CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contributing to Stryker Dashboard
+
+This is the contribution guide for Stryker Dashboard. Great to have you here! Here are a few ways you can help make this project better.
+
+## Creating issues
+
+Do you have an idea for a feature or have you found a bug? Please create an issue so we can talk about it!
+
+## Adding new features
+
+New features are welcome! Either as requests or proposals.
+
+1. Please create an issue first, so we know what to expect from you.
+2. Create a fork on your github account.
+4. Please create or edit unit tests or integration tests.
+5. Run the tests and verify they pass.
+6. Update documentation.
+
+## Maintainers
```

Comment:
Add a chapter about "Closes: #[PR number]", so people know to add this to their PR's and issues are closed properly.